### PR TITLE
Feature/adds UUID generator

### PR DESCRIPTION
**Pull request by dsmith42**
Found a useful package for KMP that should enable us to generate new UUIDs that work on iOS and Android. [uuid](https://github.com/benasher44/uuid)

Have added it to the project and implemented an example usage.

### CODE REVIEW

File: iosHackthonApp/iosHackthonApp/ContentView.swift [hunk 1]
```diff
@@ -55,6 +55,13 @@ extension ContentView {
                     self.meals = .error(error?.localizedDescription ?? "error")
                 }
             })
+            self.getNewUUID()
```
Comment (raynelle03) [inline iosHackthonApp/iosHackthonApp/ContentView.swift]: Just wondering..why are we calling it 3 times?

File: shared/build.gradle.kts [hunk 2]
```diff
@@ -28,7 +28,7 @@ kotlin {
     val coroutinesVersion = "1.3.9-native-mt"
     val serializationVersion = "1.0.0-RC"
     val ktorVersion = "1.4.0"
-    val touchlabVersion = "0.2.5"
```
Comment (Gmiller290488) [inline shared/build.gradle.kts:31]: 👍

**Review by Gmiller290488 — APPROVED** [review]

File: iosHackthonApp/iosHackthonApp/ContentView.swift [hunk 3]
```diff
@@ -55,6 +55,13 @@ extension ContentView {
...
+            self.getNewUUID()
+            self.getNewUUID()
+        }
+
+        func getNewUUID() {
+          print("UUID test: \(sdk.getUUID())")
```
Comment (Gmiller290488) [inline iosHackthonApp/iosHackthonApp/ContentView.swift]: I like the idea of showing that it works, but let's remove the calls to `getNewUUID()` :-)

**Review by raynelle03 — APPROVED** [review]

File: iosHackthonApp/iosHackthonApp/ContentView.swift (hunk 1 above)
Comment (dsmith42) [reply iosHackthonApp/iosHackthonApp/ContentView.swift]: I just wanted to check that it gave a different UUID each time.   Will remove additional calls now.

**Pull request merged**